src/phenol_verma/NotUniqueNumberArray.java

```diff
@@ -0,0 +1,98 @@
+/*
+Assignment - 85 : 23th March'2024
+Print numbers not presented uniquely from array using Array.
+Print numbers not presented uniquely from array using ArrayList.
+Print numbers not presented uniquely from array using HashMap.
+ */
+
+package phenol_verma;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class NotUniqueNumberArray {
+
+	int countFrequency(int[] arrTemp, int digit) {
+		int count = 0;
+		for (int index = 0; index < arrTemp.length; index++) {
+			if (digit == arrTemp[index]) {
+				count++;
+				arrTemp[index] = -1;
```

Comment:
not required to update the with -1, what if you have -1 in array, as here we are only counting the frequency.